src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -77,7 +86,25 @@ export class WdsDataTableProvider implements DataTableProvider {
     supportsTypeRenaming: false,
     supportsExport: false,
     supportsPointCorrection: false,
-    supportsFiltering: false
+    supportsFiltering: false,
+    supportsTabBar: false
+  }
+
+  tsvFeatures: TSVFeatures = {
+    needsTypeInput: true,
+    uploadInstructions: 'Choose the data to import below. ',
+    sampleTSVLink: 'src/../wds_template.tsv', //TODO: placeholder, does not currently work
+    invalidFormatWarning: 'Invalid format: Data does not include sys_name column.',
+    //WdsDataTableProvider and EntityServiceDataTableProvider use different variables to determine whether a file is invalid, so some are ignored here
+    isInvalid: (_0: boolean, filePresent: boolean, _2: boolean, sysNamePresent: boolean) => {
+      return !sysNamePresent && filePresent
+    },
+    disabled: (filePresent: boolean, isInvalid: boolean, uploading: boolean, recordTypePresent: boolean) => {
```

Comment:
Suggest adding return types for these functions. The return type is pretty obvious with these short functions, but it's nice to be able to glance at the signature and see the return type.

```suggestion
    disabled: (filePresent: boolean, isInvalid: boolean, uploading: boolean, recordTypePresent: boolean): boolean => {
```